### PR TITLE
[Tizen/Api] Remove Tizen dependency from CAPI set

### PR DESCRIPTION
**Pull request by again4you**
In order to support various platform APIs include Tizen, this PR removes Tizen dependency from CAPI set and reorganizes directory structure. Detailed works are as below.

* Support multiple Log environment such as Android, Tizen and Linux distro instead of dlog API.
* Use the standard error code instead of Tizen's Error code. 
* Reorganizes directory structure to support multiple platform APIs

Signed-off-by: Sangjung Woo <sangjung.woo@samsung.com>

### Related Issue
* https://github.com/nnsuite/nnstreamer/issues/1489

### Self evaluation:
1. Build test: [*]Passed [ ]Failed [ ]Skipped
2. Run test: [*]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: api/capi/include/nnstreamer.h [hunk 1]
```diff
@@ -130,11 +130,13 @@ typedef enum _ml_tensor_type_e
...
-  ML_ERROR_NOT_SUPPORTED			= TIZEN_ERROR_NOT_SUPPORTED, /**< The feature is not supported */
-  ML_ERROR_STREAMS_PIPE			= TIZEN_ERROR_STREAMS_PIPE, /**< Cannot create or access GStreamer pipeline. */
-  ML_ERROR_TRY_AGAIN				= TIZEN_ERROR_TRY_AGAIN, /**< The pipeline is not ready, yet (not negotiated, yet) */
+  ML_ERROR_NONE				= 0, /**< Success! */
+  ML_ERROR_INVALID_PARAMETER			= -EINVAL, /**< Invalid parameter */
+  ML_ERROR_STREAMS_PIPE			= -ESTRPIPE, /**< Cannot create or access GStreamer pipeline. */
+  ML_ERROR_TRY_AGAIN				= -EAGAIN, /**< The pipeline is not ready, yet (not negotiated, yet) */
+  ML_ERROR_UNKNOWN = (-1073741824LL),
+  ML_ERROR_TIMED_OUT,
+  ML_ERROR_NOT_SUPPORTED, /**< The feature is not supported */
```
Comment (myungjoo) [inline api/capi/include/nnstreamer.h]: To comply with Tizen, you may need to keep using ```TIZEN_ERROR_...``` with ```#if defined(__TIZEN__)``` for Tizen. For ```#else``` case, the current fix is great.

File: api/meson.build [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Build Tizen CAPI
```
Comment (myungjoo) [inline api/meson.build]: ```s/Tizen //```

**Comment by myungjoo** [comment]
Looks mostly good except those few items.

File: api/capi/include/nnstreamer.h (hunk 1 above)
Comment (again4you) [reply api/capi/include/nnstreamer.h]: OK. I will.

File: api/meson.build (hunk 2 above)
Comment (again4you) [reply api/meson.build]: ACK!

File: debian/rules [hunk 3]
```diff
@@ -35,7 +35,7 @@ override_dh_auto_clean:
 
 override_dh_auto_configure:
 	mkdir -p build
-	meson --buildtype=plain --prefix=/usr --sysconfdir=/etc --libdir=lib/$(DEB_HOST_MULTIARCH) --bindir=lib/nnstreamer/bin --includedir=include -Dinstall-example=true -Denable-tensorflow=$(enable_tf) build
+	meson --buildtype=plain --prefix=/usr --sysconfdir=/etc --libdir=lib/$(DEB_HOST_MULTIARCH) --bindir=lib/nnstreamer/bin --includedir=include -Dinstall-example=true -Denable-tensorflow=$(enable_tf) -Denable-capi=true build
```
Comment (jaeyun-jung) [inline debian/rules]: plz consider to set default true in meson_option and revert this change.

**Comment by again4you** [comment]
### Submit v2

* Remove unnecessary `tizen` comment in meson build script.
* Fix the build break error when building debian package
* Use TIZEN_ERROR when API is used for Tizen.

**Comment by again4you** [comment]
### Submit v3
* Tizen related Macro would be updated when CAPI is built for Non-Tizen platform.
* Update omitted `return value` in header files.
